healthCheck.ps1

```diff
@@ -0,0 +1,130 @@
+param(
+  [Parameter(Mandatory=$True,Position=1)]
+  [string]$iotHubName,
+  [Parameter(Mandatory=$True,Position=2)]
+  [string]$deviceId,
+  [Parameter(Mandatory=$True,Position=3)]
+  [string]$moduleId,
+  [Parameter(Mandatory=$True,Position=4)]
+  [string]$methodName,
+  [Parameter(Mandatory=$True,Position=5)]
+  [string]$iotHubCS,
+  [Parameter(Mandatory=$true,Position=6)]
+  [int]$maxTestWaitTimeInSeconds
+)
+
+$installed_apps = Get-CimInstance win32_product | Select-Object Name
+$az_installed = $false
+
+For ($i=0; $i -lt $installed_apps.Length; $i++) {
+$app = [string]$installed_apps[$i]
+     if ($app.contains("Microsoft") -and $app.Contains("Azure") -and $app.Contains("CLI")){    
+    $az_installed = $true
+    break}
+     }
+
+if ($az_installed -eq $false)
+{
+write-host "Azure CLI is not installed - script can not be executed"
+Exit 
+}
+
+write-host "Check if iot-extension for AZ CLI installed"
+$az_iot_ext_install_status=$(az extension show --name azure-cli-iot-ext)
+if ($az_iot_ext_install_status.Length -eq 0)
+    {
+        write-host "Add iot-extension for AZ CLI"
+        az extension add --name azure-cli-iot-ext
+    }
+
+
+# Listener wait duration
+$listenerWaitValue = 2000

```

Comment:
Do we also need to set this to maxTestWaitTimeInSeconds?

Reply:
The maxTestWaitTimeInSeconds sets the time the monitor-event commands continue execution.while this wait time is for the process itself  to  wait after the command has completed execution

Reply:
Thanks for explanation.